components/webfield/AreaChairConsole.js

```diff
@@ -624,18 +685,38 @@ const AreaChairConsole = ({ appContext }) => {
 
       <Tabs>
         <TabList>
-          <Tab id="assigned-papers" active>
+          <Tab
+            id="assigned-papers"
+            active={activeTabId === '#assigned-papers' ? true : undefined}
+            onClick={() => setActiveTabId('#assigned-papers')}
+          >
             Assigned Papers
           </Tab>
-          <Tab id="areachair-tasks" onClick={() => setActiveTabIndex(1)}>
+          {secondaryAreaChairName && (
+            <Tab
+              id="triplet-ac-assignments"
+              active={activeTabId === '#triplet-ac-assignments' ? true : undefined}
+              onClick={() => setActiveTabId('#triplet-ac-assignments')}
+            >
+              Triplet AC Assignments
```

Comment:
@celestemartinez should we call this "Secondary AC Assignments" so it is more general?

Reply:
yes, only CVPR has triplets but other venues like 3DV have used a single secondary AC